components/purchase_modal/process_payment_setup.tsx

```diff
@@ -104,7 +104,11 @@ class ProcessPaymentSetup extends React.PureComponent<Props, State> {
         } = this.props;
         const success = await addPaymentMethod((await stripe)!, billingDetails!, isDevMode);
 
-        if (!success) {
+        if (typeof success !== 'boolean' || !success) {
+            pageVisited(
+                TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
+                'pageview_payment_failed',
+            );
```

Comment:
Given that we currently can process `page` events in our data pipelines, let's change this event to be a `track` event instead of `page`